backend-agent/cli.py

```diff
@@ -379,6 +379,62 @@ def run(args):
               file=sys.stderr)
 
 
+def run_all_logic(args):
+    """Run all LLM attacks with specified target and evaluation models."""
+    # Load JSON spec
+    if not args.file:
+        print(
+            'No file given. Enter specification using stdin.',
+            file=sys.stderr
+        )
+        input_data = ''
+        for line in sys.stdin:
+            input_data += line
+            if line == '\n':
+                break
+        if not input_data:
+            print(
+                'Specify a path to a JSON spec or provide it via stdin.',
```

Comment:
let's not offer the possibility to provide a json in stdin. It's counterintuitive before a mess to manage. If users want to provide their own spec, they will overwrite the default.json